src/components/resourceTable.js

```diff
@@ -0,0 +1,357 @@
+import { DEPLOYMENT_STATUS, RESOURCE_TABLE_TYPE } from "../constants";
+import { getStackResources } from "../services/stackResources";
+
+const contrib = require("blessed-contrib");
+const open = require("open");
+const { exec } = require("child_process");
+const emoji = require("node-emoji");
+const moment = require("moment");
+
+class ResourceTable {
+    constructor(application, program, provider, slsDevToolsConfig, profile, location, cloudformation, lambda) {
+        this.application = application;
+        this.lambdaFunctions = {};
+        this.latestLambdaFunctionsUpdateTimestamp = -1;
+        this.program = program;
+        this.cloudformation = cloudformation;
+        this.table = this.generateLambdaTable();
+        this.funcName = null;
+        this.fullFuncName = null;
+        this.table.rows.on("select", (item) => {
```

Comment:
Separate into a separate function, not in the constructor

Reply:
How does this differ from getCurrentlySelectedLambdaName ?

Reply:
it is different from `getCurrentlySelectedLambdaName` because `getCurrentlySelectedLambdaName` returns the lambda which is on hover. I probably should call it `getCurrentlyHoveredLambda`